kolibri/plugins/setup_wizard/api.py

```diff
@@ -173,3 +173,26 @@ class SetupWizardSoUDTaskView(TasksViewSet):
     """
 
     permission_classes = [HasPermissionDuringSetup | HasPermissionDuringLODSetup]
+
+
+class SetupWizardRestartZeroconf(ViewSet):
+    """
+    An utility endpoint to restart zeroconf after setup is finished
+    in case this is a SoUD
+    """
+
+    permission_classes = [HasPermissionDuringSetup | HasPermissionDuringLODSetup]
+
+    @decorators.action(methods=["post"], detail=False)
+    def restart(self, request):
+        import logging
```

Comment:
Any reason not to do these imports and logging configuration in module scope?

Reply:
just to avoid doing the import if the wizard does the other two kinds of setup instead of the SoUD provisioning